apiary.apib

```diff
@@ -187,30 +187,30 @@ Any response that does not provide a ```200 OK``` response code should be treate
 + `AncestorNamePath` (string, required) - The anscestor name path.
 
 ## Cost (object)
-+ `labor` (number, required) - The sum of labor costs.
-+ `purchase` (number, required) - The sum of purchase costs.
-+ `total` (number, required) - The total sum of all costs.
++ `labor` (number, required) - #INT The sum of labor costs.
++ `purchase` (number, required) - #INT The sum of purchase costs.
++ `total` (number, required) - #INT The total sum of all costs.
 
 ## Count (object)
-+ `workOrders` (number, required) - The number of workorders.
-+ `laborHours` (number, required) - The number of labor hours.
++ `workOrders` (number, required) - #INT The number of workorders.
++ `laborHours` (number, required) - #INT The number of labor hours.
```

Comment:
I would leave these as decimals too unless they are set as integers in the API/DB.  Time is usually tracked in half to quarter hour increments